simple_regression_test.py

```diff
@@ -64,6 +81,9 @@ def test_simple_regression(self):
                          mode='mixed')
         mixed_results = self.get_stress_results(queue=stress_queue)
 
+        display_results(write_results)
+        display_results(read_results)
+        display_results(mixed_results)
```

Comment:
Overall, nice idea. One thing that we need to think about is a criteria to automatically fail the test in case differences are found between each result set. For example, if there's a drastic performance drop between write and mixed results (say, over 10%), fail the test or something similar.